docs/releases/dev.rst

```diff
@@ -22,7 +22,7 @@ Major Changes
 
 - Updated translations.
 - Added django-transaction-hooks
-
+- Changed user delete behaviour
```

Comment:
Just noticed that this is expanded below. But the blank line concern still applies.